gemfire-core/src/main/java/com/gemstone/gemfire/internal/snappy/StoreCallbacks.java

```diff
@@ -32,4 +32,5 @@
   List<String> getInternalTableSchemas();
   public int getHashCodeSnappy(Object dvd);
   public int getHashCodeSnappy(Object dvds[]);
+  public Boolean haveRegisteredExternalStore(String tableName);
```

Comment:
Return type can better be primitive boolean. See comment in other snappydata PR.